packages/theme/src/cli/services/dev.ts

```diff
@@ -42,31 +43,40 @@ export async function dev(options: DevOptions) {
     return
   }
 
-  if (options.flagsToPass.includes('--poll')) {
-    renderWarning({
-      body: 'The CLI flag --[flag-name] is now deprecated and will be removed in future releases. It is no longer necessary with the new implementation. Please update your usage accordingly.',
-    })
-  }
+  if (options['dev-preview']) {
```

Comment:
I don't think we need this condition, because if [the code is reaching this point](https://github.com/Shopify/cli/blob/61dfa17e6f2091d3eebf301439dd523139f844f3/packages/theme/src/cli/services/dev.ts#L37-L40), it's already running the new implementation :)

Reply:
(now, that I'm reading a bit more, I think we maybe have conflict issue on this method)